script/export_for_foi/export_mappings.sql

```diff
@@ -0,0 +1,6 @@
+COPY (
+  SELECT sites.abbr, path, new_url, suggested_url, archive_url,
+    CASE type WHEN 'unresolved' THEN 410 WHEN 'archive' THEN 410 WHEN 'redirect' THEN 301 END AS http_status
+    FROM mappings
+    INNER JOIN sites ON mappings.site_id = sites.id
```

Comment:
We should exclude mappings for sites with a global status with a `WHERE` clause.